betty/gramps/loader.py

```diff
@@ -148,30 +148,29 @@ async def load_tree(self, tree: ElementTree.ElementTree, gramps_tree_directory_p
         database = self._tree.getroot()
 
         self._load_notes(database)
-        # @todo Localize all log messages
-        logger.info(f'Loaded {self._added_entity_counts[Note]} notes.')
+        logger.info(self._localizer._(f'Loaded {self._added_entity_counts[Note]} notes.'))
```

Comment:
The way string translation works, is that all strings MUST be passed in to the gettext methods as static string literals. Literals because of the way they are extracted, static because the strings themselves act as unique keys.

In short, that means any formatting must be done after translation. The solution is straightforward: use regular strings instead of f-strings, and call `.format()` on the translated string to inject the variables.